test/specs/routing.test.js

```diff
@@ -291,14 +299,26 @@ module.exports = fetchMock => {
 
 			describe('query strings', () => {
 				it('can match a query string', async () => {
-					fm.mock('http://it.at.there/', 200, { query: { a: 'b' } }).catch();
```

Comment:
Prettier wouldn't let me write it like this anymore, so I decided to normalize the two.

Reply:
:+1: